main.py

```diff
@@ -1,10 +1,36 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players = json.load(players_file)
+
+        for player_name, player_info in players.items():
+            race_data = player_info["race"]
+            race, created = Race.objects.get_or_create(
+                name=race_data["name"],
+                defaults={"description": race_data["description"]})
+            for skill_data in race_data["skills"]:
+                Skill.objects.get_or_create(
+                    name=skill_data["name"],
+                    defaults={"bonus": skill_data["bonus"], "race": race})
+            guild = None
+            if player_info["guild"]:
```

Comment:
Ensure that 'player_info["guild"]' is checked for None or empty values correctly. If 'guild' can be an empty string or another falsy value, consider using a more explicit check like 'if player_info.get("guild") is not None:' to avoid potential issues.